src/app/hlarp_cli.ml

```diff
@@ -60,10 +60,25 @@ let () =
   in
   let athlates =
     let tool = "athlates" in
-    Term.(const (fun dir ->
-            Athlates.scan_directory dir
+    let equal_pairs_flag =
+      Arg.(value
+           & vflag `MostLikelyPair
+              [ `MostLikelyPair, info ["most_likely_pair"]
+                  ~doc:"For each typing file report the most frequent two alleles (default)."
```

Comment:
ITOCTYDK (in the off-chance that you don't know) FWIW  Cmdliner will already note that the `MostLikelyPair is the default (by saying absent=most_likely_pair, which isn't all that great, so the additional note that it's the default in the docstring is still nice).